### PR TITLE
Issue #151: Update layout of  GUI interview panel

### LINKED ISSUE #151: Update GUI right panel to display confirmed upcoming interviews

### LINKED ISSUE #171: Improve code quality for predicate classes
Change the accessor of field from `private` to `protected`, and edit constructor body.

### PR DESCRIPTION
**Pull request by charmainehly**
**This PR includes:**
- update to the bottom right panel for interview schedule to display necessary information
- added padding between the cells plus some cosmetic changes w.r.t. the colours.
- minor fixes to predicate classes to improve code quality (i.e. w.r.t. accessors in the parent abstract class)
- fix checkstyle errors _(CI workflow wouldn't pass without me fixing the trailing whitespace errors_

<br>**Expected layout from this PR (upon scheduling some interviews in your local machine):**
Screenshot 2022-03-27 at 1 45 14 PM

_Note: It doesn't really look that great, but was hoping to add a little bit of colour to differentiate the panels, feel free to suggest colour changes/layout adjustments. Or if you would like to modify the layout as well feel free to do so after merging!_

Closes #151 
Closes #171

### CODE REVIEW

**Comment by charmainehly** [comment]
Note: failing codecov as changes relate to UI!

**Review by tiewweijian — APPROVED** [review]
Wow I love the design for this !! Maybe the next iteration (or whoever's free), should we also have the same template for the left panel too? But other than that, LGTM !!

File: src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -10,7 +10,6 @@
  * Tests that a {@code Candidate} matches any of the keywords given.
  */
 public class CandidateContainsKeywordsPredicate extends ContainsKeywordsPredicate implements Predicate<Candidate> {
-    private final List<String> keywords;
```
Comment (domlimm) [inline src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java:13]: Oops @charmainehly, I might have to fix this for the `availability` to follow yours. Will KIV it on my end and discuss if needed.

File: src/main/java/seedu/address/model/interview/predicate/TodayWithinTimePeriodPredicate.java [hunk 2]
```diff
@@ -9,7 +9,6 @@
  * Tests that a {@code Interview}'s date and time is within the same day.
  */
 public class TodayWithinTimePeriodPredicate extends WithinTimePeriodPredicate implements Predicate<Interview> {
```
Comment (domlimm) [inline src/main/java/seedu/address/model/interview/predicate/TodayWithinTimePeriodPredicate.java:11]: Nice job following polymorphism, inheritance and abstraction!

File: src/main/resources/view/DarkTheme.css [hunk 3]
```diff
@@ -430,3 +430,41 @@
 .keyAttributesVBox {
     -fx-padding: 10 10 10 0;
 }
+
+
+// ************* interview panel css *************
```
Comment (domlimm) [inline src/main/resources/view/DarkTheme.css:435]: DOPE! 💯

**Review by domlimm — APPROVED** [review]
No issues at all. Great job on the UI @charmainehly! Well done!

**Pull request merged**